keyvi/include/keyvi/dictionary/matching/fuzzy_matching.h

```diff
@@ -93,7 +93,7 @@ class FuzzyMatching final {
         continue;
       }
 
-      if (traverser_ptr_->IsFinalState() && metric_ptr_->GetScore() <= max_edit_distance_) {
+      if (traverser_ptr_->IsFinalState() && metric_ptr_->GetScore() <= static_cast<int>(max_edit_distance_)) {
```

Comment:
@amit-cliqz do we still need this type casting ?

Reply:
@narekgharibyan yes, unless we refactor Distance matrix and GetScore function. I have a TODO for that.